assignments/lambda-classes.js

```diff
@@ -1 +1,41 @@
 // CODE here for your Lambda Classes
+
+const Person = new Instructor ({
+    name: 'Abby',
+    age: 26,
+    location:'New York',
+    gender:'female',
+    
+)}
+
+class Instructor extends Person{
+    constructor(props) {
+        super(props);
+        this.name = props.name;
+        this.age = props.age;
+        this.location = props.loctaion;
+        this.gender = props.gender;
+        specialty: 'Front-end';
```

Comment:
make sure to place this to the new object literal you would be creating if you do 
```
const testInstructor = new Instructor ({
    // specialty goes here
})
```